src/main/resources/META-INF/services/org.apache.pirk.responder.wideskies.spi.ResponderPlugin

```diff
@@ -0,0 +1,5 @@
+org.apache.pirk.responder.wideskies.mapreduce.MapReduceResponder
```

Comment:
If not plug-ins then can you explain the way you see it working?

I had a guess on [this thread](https://lists.apache.org/thread.html/2f2c72c264a43b404416c45ad978aba554e7569a4787b62f45c2802d@%3Cdev.pirk.apache.org%3E), but didn't see a reply.